lib/node/nodes/gravity.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+var Node = require('../node');
+
+var TYPE = 'gravity';
+
+var PARAMS = {
+    source: Node.PARAM.NODE(Node.GEOMETRY.ANY), // "populated places"
+    target: Node.PARAM.NODE(Node.GEOMETRY.POINT), // "malls"
+    weightColunn: Node.PARAM.STRING(),
+    weightThreshold: Node.PARAM.NULLABLE(Node.PARAM.NUMBER()),
+    popColumn: Node.PARAM.STRING(),
+    maxDistance: Node.PARAM.NUMBER(), // meters
+    targetID: Node.PARAM.NUMBER() // the "mall" to be studied
+};
+
+var Gravity = Node.create(TYPE, PARAMS, {cache: true, version: 1});
+
+module.exports = Gravity;
+module.exports.TYPE = TYPE;
+module.exports.PARAMS = PARAMS;
+
+var gravityTemplate = Node.template([
+    'WITH ',
+    ' raw_s as({{=it.sourceQuery}}),',
+    ' raw_t as({{=it.targetQuery}})',
+    ' s as(',
```

Comment:
Fixed, added the commas. I feel a bit lost about the acceptance tests you use in camshaft, Id' like to add tests like I'm adding in crankshaft stuff....